### PR TITLE
[new release] thrift (0.10.0)

### PR DESCRIPTION
**Pull request by vbmithr**
OCaml bindings for the Apache Thrift RPC system

- Project page: <a href="https://github.com/vbmithr/ocaml-thrift-lib">https://github.com/vbmithr/ocaml-thrift-lib</a>
- Documentation: <a href="https://vbmithr.github.io/ocaml-thrift-lib/doc">https://vbmithr.github.io/ocaml-thrift-lib/doc</a>

##### CHANGES:

* Sync with upstream
* Make the code safe-string ready (@c-cube)
* build: switch to dune (@c-cube)

### CODE REVIEW

File: packages/thrift/thrift.0.10.0/opam [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+doc: "https://vbmithr.github.io/ocaml-thrift-lib/doc"
+depends: [
+  "ocaml" { >= "4.03.0" }
+  "dune" { >= "1.1" }
+]
+depexts: [
+  ["thrift"] {os = "archlinux"}
+  ["thrift-compiler"] {os-family = "debian"}
+  ["devel/thrift"] {os-family = "bsd"}
+]
```
Comment (kit-ty-kate) [inline packages/thrift/thrift.0.10.0/opam]: The depexts don't seem to be strictly needed. I could only find use in the examples. Is this normal?

Comment (vbmithr) [reply packages/thrift/thrift.0.10.0/opam]: You are right, yes. I think it was necessary for CI to run. Anyway, it's probably not a bad idea to have thrift installed to use the library…

**Comment by kit-ty-kate** [comment]
I guess it can always be modified later. Thanks!

**Comment by kit-ty-kate** [comment]
Could you also return the fix upstream?

**Comment by vbmithr** [comment]
Yes, will do it now.

**Pull request merged**